tests/ha/crm_mon.pm

```diff
@@ -0,0 +1,15 @@
+use base "hacluster";
+use testapi;
+use autotest;
+use lockapi;
+
+sub run() {
+    type_string "crm_mon -1\n";
+    assert_screen "ha-crm-mon-" . get_var("CLUSTERNAME"), 20;
```

Comment:
low timeout values have a problem sometimes, just use the default and don't specify a timeout